V2/tests/api/azure/activity_azure_endpoints_test.py

```diff
@@ -13,8 +15,8 @@ def test__activity_azure_endpoint__returns_all_activities(
 ):
     existent_activities = [activity_factory(), activity_factory()]
     inserted_activities = [
-        insert_activity(existent_activities[0], test_db).__dict__,
-        insert_activity(existent_activities[1], test_db).__dict__
+        parse_status(insert_activity(existent_activities[0], test_db).__dict__),
```

Comment:
I think we could place this function in a more global folder that can be used from several modules.